daemons/controld/controld_messages.c

```diff
@@ -439,6 +440,18 @@ relay_message(xmlNode * msg, gboolean originated_locally)
         }
     }
 
+    // If the DC is not yet selected
+    if ((strcmp(task, CRM_OP_PING) == 0) && (controld_globals.dc_name == NULL)) {
```

Comment:
better to use `pcmk__str_eq()` instead of `strcmp()` in case task is NULL

Reply:
Yap. Changed.